framework/src/wifi_manager/wifi_manager.c

```diff
@@ -196,7 +190,7 @@ static pthread_cond_t g_disconn_signal = PTHREAD_COND_INITIALIZER;
 		g_manager_info.disconn_substate = WIFIMGR_DISCONN_NONE;     \
 		pthread_cond_signal(&g_disconn_signal);                     \
 		pthread_mutex_unlock(&g_disconn_lock);                      \
-		nvdbg("[WM] T%d received disconnect callback\n", getpid()); \
+		ndbg("[WM] T%d received disconnect callback\n", getpid()); \
```

Comment:
Is this an error situation so that you changed nvdbg to ndbg..?

Reply:
This is not error case but related with timing issue. So considering the importance, made it to ndbg. Thanks for the comment.